website/source/docs/enterprise/index.html.md

```diff
@@ -8,19 +8,19 @@ description: |-
 
 # Consul Enterprise
 
-Consul Enterprise simplifies operations by automating workflows. It adds support
-for microservices deployments across complex network topologies. It also
-increases both scalability and resilience. If you have already purchased Consul Enterprise, please see the [licensing section](#licensing)
-below.
+[Consul Enterprise](https://www.hashicorp.com/consul.html) focuses on the concerns of operating Consul at 
+organizational scale by automating common operator workflows. It introduces capabilities for expanding 
+performance scalability, resiliency and cluster lifecycle. It also adds support for application and service 
+architectures across complex network topologies. If you have already purchased Consul Enterprise, please 
+see the [licensing section](#licensing) below.
 
 Features include:
 
 - [Automated Backups](/docs/enterprise/backups/index.html)
 - [Automated Upgrades](/docs/enterprise/upgrades/index.html)
```

Comment:
@codyde Do you have an asana to track upcoming changes? @danielehc is working on new guides for Redundancy zones and automated backups.

Reply:
Upcoming changes as in future downstream documentation changes? I don't have one immediately. We have one for the future state updating of .io docs in general - but I'll look at creating another one for the next round of changes to include.